v3/src/components/graph/models/graph-content-model.ts

```diff
@@ -351,15 +350,108 @@ export const GraphContentModel = DataDisplayContentModel
         : casePrimaryValue
       const firstCount = legendAttrID ? totalCases : casesInSubPlot
       const secondCount = legendAttrID ? casesInSubPlot : totalCases
-
+      const percent = float(firstCount / secondCount * 100)
```

Comment:
I would write this as:
```suggestion
      const percent = float(100 * firstCount / secondCount)
```
to avoid the reader mistakenly thinking that the 100 is applied to the denominator (which it isn't, but visually it looks like it might be).